charts/karavi-observability/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "1.2.0"
+appVersion: "1.2.1"
```

Comment:
this will be 1.3.0, every new release updates the minor version. Only patch releases increment the last digit.